components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterEditor.scala

```diff
@@ -4,23 +4,18 @@ import io.circe.{Decoder, Encoder, Json}
 import io.circe.generic.JsonCodec
 import io.circe.generic.extras.ConfiguredJsonCodec
 import pl.touk.nussknacker.engine.api.CirceUtil._
-import pl.touk.nussknacker.engine.api.editor.DualEditorMode
 
 import java.time.temporal.ChronoUnit
 import scala.util.Try
 
 @ConfiguredJsonCodec sealed trait ParameterEditor
 
-case object RawParameterEditor extends ParameterEditor
```

Comment:
Let's say that I want to add another editor, should I extend `SimpleParameterEditor` or just `ParameterEditor`?

Reply:
Done - removed `SimpleParameterEditor`, but I added `StaticParameterEditor` which will be available only for scenario properties and action parameters